<launch>
    <arg name="camera_namespace" default="camera_center"/>

    <node name="cone_bottom_detector" pkg="rr_iarrc" type="cone_bottom_detector"
          output="screen" ns="$(arg camera_namespace)" required="true">
        <!--HSV color threshold for orange cones-->
        <param name="orange_low_H" type="int" value="0" />
        <param name="orange_high_H" type="int" value="25" />
        <param name="orange_low_S" type="int" value="130" />
        <param name="orange_low_V" type="int" value="75" />

        <!--The larger the height the lower it will be on the screen (Image Height = 1280)-->
        <param name="blockSky_height" type="int" value="0" />
        <param name="blockWheels_height" type="int" value="750" />
        <param name="blockBumper_height" type="int" value="720" />

        <param name="subscription_node" type="string" value="image_color_rect"/>
    </node>

</launch>
